src/api/goods.js

```diff
@@ -6,6 +6,10 @@ export function getAll() {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = () => getAll()
+  .then(goods => goods
+    .sort((good1, good2) => good1.name.localeCompare(good2))
```

Comment:
the list is not sorted now

```suggestion
    .sort((good1, good2) => good1.name.localeCompare(good2.name))
```

Reply:
ну это просто кошмар какой-то(((